src/components/List/specs/List.spec.jsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import List, { ListItem, ListSubheader } from '../src/index';
+import { HouseIcon, TrashIcon } from '../../Icons';
+
+describe(`List tests`, () => {
+  it('has clickable listitems', () => {
+    const props = {
+      onClick: () => {
+        console.log('Clicked');
+      },
+    };
+
+    const spy = cy.spy(props, 'onClick').as('List item click');
```

Comment:
I noticed we're testing keyboard usage in some components. Might be a good idea to test it here too

Reply:
I added some keyboard testing, but quick sidenote: almost all of our keyboard `{enter}` tests currently add absolutely nothing due to https://github.com/nl-design-system/denhaag/issues/428.